app/Renderers/Layouts/Location/Character.php

```diff
@@ -32,9 +32,16 @@ public function columns(): array
             ],
             'families' => [
                 'label' => Module::plural(config('entities.ids.family'), 'entities.families'),
-                'class' => self::ONLY_DESKTOP,
-                'render' => Standard::ENTITYLIST,
-                'with' => 'families',
+                'render' => function ($model) {
```

Comment:
Can't you just do `'with' => 'families'` and skip the whole custom rendering?

Reply:
Done that way the privacy functions dont work properly, it was done like that at first for character races and it caused issues.